src/cirq_iqm/iqm_sampler.py

```diff
@@ -145,6 +150,7 @@ def _send_circuits(
         circuits: list[cirq.Circuit],
         calibration_set_id: Optional[UUID],
         repetitions: int = 1,
+        circuit_duration_check: bool = True,
```

Comment:
Do we need to have this as a parameter, since it's already an instance attribute for the class?

Reply:
I guess same can be asked also for `calibration_set_id`.

Reply:
Yup, removed.